src/containers/explore/EntityNeighborsTable.js

```diff
@@ -0,0 +1,177 @@
+/*
+ * @flow
+ */
+
+import React, { useEffect, useMemo, useState } from 'react';
+
+import {
+  List,
+  Map,
+  Set,
+  get
+} from 'immutable';
+import { Models } from 'lattice';
+import {
+  CardSegment,
+  Colors,
+  PaginationToolbar,
+  Table,
+} from 'lattice-ui-kit';
+import { ReduxUtils, useRequestState } from 'lattice-utils';
+import { useDispatch, useSelector } from 'react-redux';
+import { RequestStates } from 'redux-reqseq';
+import type { UUID } from 'lattice';
+import type { RequestState } from 'redux-reqseq';
+
+import { EntityDataRow } from './components';
+
+import { TableCardSegment } from '../../components';
+import { FETCH_ENTITY_SET_DATA, fetchEntitySetData } from '../../core/data/actions';
+import { useEntityTypePropertyTypes } from '../../core/edm/utils';
+import { resetRequestStates } from '../../core/redux/actions';
+import { DATA } from '../../core/redux/constants';
+import { selectOrgEntitySetData } from '../../core/redux/selectors';
+import { MAX_HITS_10 } from '../../core/search/constants';
+
+const { PropertyType } = Models;
+const { isPending } = ReduxUtils;
+const { PURPLE } = Colors;
+
+type Props = {
+  associationDataSet :Map,
+  dataSet :Map;
+  isModal :boolean;
+  neighbors :List<Map>;
+  organizationId :UUID;
+};
+
+const EntityNeighborsTable = ({
+  associationDataSet,
+  dataSet,
+  isModal,
+  neighbors,
+  organizationId
+} :Props) => {
+  const associationDataSetId :UUID = get(associationDataSet, 'id');
+  const dataSetId :UUID = get(dataSet, 'id');
+
+  /*
+   * NOTE: performance issue against prod
+   * /explore/entityData/77f1b8d0-9c75-4e52-8176-eb6913a74669/f7cc0000-0000-0000-8000-0000000070ab
+   */
```

Comment:
Are there any updates on this issue?

Reply:
this comment was pulled in from another file. removed.